src/utf8.rs

```diff
@@ -120,7 +120,7 @@ impl Validator {
 
             self.reset();
 
-            unsafe { input.get_unchecked(bytes_to_copy..) }
+            &input[bytes_to_copy..]
```

Comment:
`bytes_to_copy` is computed above based on `input.len()`

Reply:
This is also fine, see https://godbolt.org/z/67jsshs95